resources/js/common/components/AppProviders/AppProviders.tsx

```diff
@@ -0,0 +1,18 @@
+import { LaravelReactI18nProvider } from 'laravel-react-i18n';
+import type { FC, ReactNode } from 'react';
+
+interface AppProvidersProps {
+  children: ReactNode;
+}
+
+export const AppProviders: FC<AppProvidersProps> = ({ children }) => {
+  return (
+    <LaravelReactI18nProvider
+      locale="en_US"
```

Comment:
Eventually, some day in the far future, this will be dynamic and populated during the server render cycle.